command/meta.go

```diff
@@ -67,7 +71,15 @@ func (m *Meta) Client() (*api.Client, error) {
 	if m.ForceAddress != "" {
 		config.Address = m.ForceAddress
 	}
-
+	if v := os.Getenv(EnvVaultCACert); v != "" {
+		m.flagCACert = v
+	}
+	if v := os.Getenv(EnvVaultCAPath); v != "" {
+		m.flagCAPath = v
+	}
+	if v := os.Getenv(EnvVaultInsecure); strings.ToLower(v) == "true" || strings.ToLower(v) == "yes" {
```

Comment:
I'll defer to @mitchellh and @armon, but I would prefer to stay consistent using Go's stdlib.